patches/chrome-browser-notifications-notification_platform_bridge_android.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/notifications/notification_platform_bridge_android.cc b/chrome/browser/notifications/notification_platform_bridge_android.cc
+index 54ac388ee6416a8e5c2e5d273d64fa8fc5958bd8..ec19878355b013ea3d1b73335c7fe131803850ee 100644
+--- a/chrome/browser/notifications/notification_platform_bridge_android.cc
++++ b/chrome/browser/notifications/notification_platform_bridge_android.cc
+@@ -321,6 +321,7 @@ void NotificationPlatformBridgeAndroid::Display(
+ void NotificationPlatformBridgeAndroid::Close(
+     Profile* profile,
+     const std::string& notification_id) {
++  BRAVE_MAYBE_CLOSE_AD_NOTIFICATION;
```

Comment:
I this there is a better way to do this because we should haven't brave ads specific code in the notification bridge, but for future reference this define should be `BRAVE_CLOSE`